lib/redact-secrets/index.js

```diff
@@ -0,0 +1,45 @@
+'use strict'
+// This module is a fork of
+// https://github.com/watson/redact-secrets/blob/cf8f95aaa343a85978c09222096fde5f4d608aea/index.js
```

Comment:
```suggestion
// https://github.com/watson/redact-secrets/blob/v1.0.0/index.js
```

*Could* use the tag name here.

Reply:
Ah, yes.   This would be better.  I'm addicted to the `y` key: https://docs.github.com/en/free-pro-team@latest/github/managing-files-in-a-repository/getting-permanent-links-to-files. Updating this.

Reply:
> addicted to the `y` key

ditto